### PR TITLE
add 4 entities

### PR DESCRIPTION
**Pull request by OleksandrOpanasiuk**

### CODE REVIEW

**Review by FedosovMax — APPROVED** [review]

File: src/main/java/com/parsing/model/LotResult.java [hunk 1]
```diff
@@ -34,19 +27,42 @@ public class LotResult {
...
+    @ManyToMany
+    private List<Participant> participants;
+
+    @Column(name = "lot_url")
+    private String lotURL;
+
+    @Column(name = "pdf_url")
+    private String pdfURL;
+
+    @OneToMany(mappedBy = "lotInfo", cascade = CascadeType.ALL)
```
Comment (YuriiZaripa) [inline src/main/java/com/parsing/model/LotResult.java]: I'm not sure you have to add in LotInfo information regarding LotInfoItem. LotInfoItem only for LotInfo.

File: src/main/java/com/parsing/model/LotResult.java [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
-import jakarta.persistence.CascadeType;
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.EnumType;
-import jakarta.persistence.Enumerated;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.Id;
-import jakarta.persistence.OneToOne;
-import jakarta.persistence.Table;
+import jakarta.persistence.*;
```
Comment (YuriiZaripa) [inline src/main/java/com/parsing/model/LotResult.java]: Don't use a wildcard for import.

Comment (OleksandrOpanasiuk) [reply src/main/java/com/parsing/model/LotResult.java]: Done!

File: src/main/java/com/parsing/model/LotResult.java (hunk 1 above)
Comment (OleksandrOpanasiuk) [reply src/main/java/com/parsing/model/LotResult.java]: Done.

**Pull request merged**